app/assets/stylesheets/application.scss

```diff
@@ -88,4 +88,16 @@ main {
 .currency-symbol {
   font-size: 25px;
   vertical-align: super;
-}
\ No newline at end of file
+}
+
+.pac-container {
+  z-index: 20;
+}
+
+.modal {
+  z-index: 20;   
+}
```

Comment:
Are all these really needed? If so, might want to leave a comment in the code as to why. Also, you might want to combine the two above with:
```scss
.pac-container,
.modal {
  z-index: 20;
}
```
Assuming they're related somehow